source/AI.cpp

```diff
@@ -442,9 +442,14 @@ void AI::UpdateKeys(PlayerInfo &player, Command &activeCommands)
 		newOrders.target = target;
 		IssueOrders(player, newOrders, "focusing fire on \"" + target->Name() + "\".");
 	}
+	if(activeCommands.Has(Command::HOLD_FIRE))
+	{
+		newOrders.type = Orders::HOLD_FIRE;
+		IssueOrders(player, newOrders, "holding fire.");
+	}
```

Comment:
Why insert the hold fire command between these two fight commands?

Reply:
~~conflict resolving online thats why~~